README.md

```diff
@@ -1,5 +1,22 @@
-# crowdin-gateway
-Gateway for integrating with Crowdin for translation of content
+# Crowdin Gateway
 
+# Usage
+Gateway for integrating with Crowdin for translation of content.
 
+* The `content` of the request is assumed to be valid html.
+* The `metaData` of the request is assumed to be valid json.
 
+To interact with the API, you need valid security credentials; see [Access Tokens usage](https://github.com/NDLANO/auth/blob/master/README.md).
+
+For a more detailed documentation of the API, please refer to the [API documentation](https://staging.api.ndla.no).
```

Comment:
Heh...det var egentlig ikke gjennomtenkt. Det var copy-paste fra en annen readme. For min del trenger vi ingen diskusjon. Jeg synes gjerne vi kan lenke rett til https://api.ndla.no (bortsett fra at flesteparten av de apiene som vi nå dokumenterer ikke er i prod)